.github/workflows/generate-allure-report.yml

```diff
@@ -32,60 +36,50 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Setup CI
-        uses: input-output-hk/catalyst-ci/actions/setup@master
+      - name: Install Forge
+        uses: input-output-hk/catalyst-forge/actions/install@ci/v1.1.0
+        if: ${{ inputs.forge_version != 'local' }}
+        with:
+          version: ${{ inputs.forge_version }}
+
+      - name: Install Local Forge
+        uses: input-output-hk/catalyst-forge/actions/install-local@ci/v1.1.0
+        if: ${{ inputs.forge_version == 'local' }}
```

Comment:
I'm thinking we can avoid running the allure report if the ci is runned locally?what do you think?